docs/user-guide/deployments/deploy-on-kubernetes/getting-started.md

```diff
@@ -300,7 +300,40 @@ The greptimedb-cluster is starting, use `kubectl get pods -n default` to check i
 ```
 </details>
 
-You can check the status of the GreptimeDB cluster:
+When the `monitoring` option is enabled, the Helm chart configuration will use `mycluster` and `default` as the default cluster name and namespace.
+
+If you want to modify the default cluster name and namespace, you can create a `values.yaml` file like this:
+
+```yaml
+grafana:
+  datasources:
+    datasources.yaml:
+      datasources:
+        - name: greptimedb-metrics
+          type: prometheus
+          url: http://<cluster>-monitor-standalone.<namespace>.svc.cluster.local:4000/v1/prometheus
+          access: proxy
+          isDefault: true
+
+        - name: greptimedb-logs
+          type: mysql
+          url: <cluster>-monitor-standalone.<namespace>.svc.cluster.local:4002
+          access: proxy
+          database: public
+```
+
```

Comment:
please explain the code above that grafana monitoring the two clusters greptimedb-metrics and greptimedb-logs. please also explain what is the `type`.
Why the greptimedb-logs needs to specify the database public while greptimedb-metrics doesn't?

Reply:
I added some descriptions for the configuration. Because the Prometheus protocol doesn't have the database concept, it doesn't need to specify the database.